src/main/java/org/cascadebot/cascadebot/UnicodeConstants.java

```diff
@@ -34,4 +34,7 @@
     public static final String FROWNING = "\uD83D\uDE26"; // 😦
     public static final String THINKING = "\uD83E\uDD14"; // 🤔
 
+    public static final String ONE = "\uD83C\udde6";
+    public static final String TWO = "\uD83C\udde7";
```

Comment:
I'm gonna go and say add all the numbers. Also, `\uD83C\udde6` is not `1`? It's 🇦

Reply:
Whops